test/utils/EIP712Utils.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  MessageTypeProperty,
+  MessageTypes,
+  signTypedData,
+  SignTypedDataVersion,
+  TypedMessage,
+} from '@metamask/eth-sig-util';
+
+import {
+  EnvelopingTypes,
+  IForwarder,
+} from '../../typechain-types/@rsksmart/rif-relay-contracts/contracts/RelayHub';
+
+export type ForwardRequest = IForwarder.ForwardRequestStruct;
+export type RelayData = EnvelopingTypes.RelayDataStruct;
+export type RelayRequest = EnvelopingTypes.RelayRequestStruct;
+export type DeployRequest = EnvelopingTypes.DeployRequestStruct;
+
+const eIP712DomainType: MessageTypeProperty[] = [
```

Comment:
should be better to re-use the types from here https://github.com/rsksmart/rif-relay-client/blob/develop/src/typedRequestData.utils.ts#L4 and just wrap it up in the new types?

Reply:
I actually did this here: https://github.com/rsksmart/rif-relay/blob/develop/test/EIP712utils.ts